docs/clients/brushless_drive.rst

```diff
@@ -105,72 +105,116 @@ Message Table
+| 50     | supply_current_limit_enable   | get, set, save | uint8     | bool     |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 51     | regen_limiting                | get            | uint8     | bool     |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 52     | regen_limit_adjust            | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 53     | motoring_limiting             | get            | uint8     | bool     |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 54     | motoring_limit_adjust         | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 55     | regen_limit_kp                | get, set, save | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 56     | regen_limit_ki                | get, set, save | float     |          |                                                                                                                                      |      |
```

Comment:
Description: "The I gain for the closed loop controller of the regeneration dynamic supply current limiter."

Units are (V*s)/A